packages/banner/.babelrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
What's this about?

Reply:
I found Jest to be confused without it. Somehow this `.babelrc` file gets picked up...

```
 FAIL  packages/banner/src/BannerPresenter/BannerPresenter.test.js
  ● Test suite failed to run

    /Users/efung/Projects/Autodesk/hig/packages/banner/src/BannerPresenter/BannerPresenter.test.js: Unexpected token (31:17)
        29 |       props: {
        30 |         dismissButtonTitle: "boom",
      > 31 |         actions: <span>foo</span>
           |                  ^
        32 |       }
        33 |     },
        34 |     {
```